vault/plugin_catalog.go

```diff
@@ -237,8 +243,6 @@ func (c *PluginCatalog) getPluginClient(ctx context.Context, sys pluginutil.Runn
 
 	pc.protocol = rpcClient
```

Comment:
Probably here we want to wrap the rpcClient in our own abstraction over it that does the grpc Invoke call injection?

Reply:
resolved in: https://github.com/hashicorp/vault/pull/13896/commits/e9da3b4e34f8ac7efe9d70b6a5c1f8da6c3facc8